### PR TITLE
Add base docker and docker-compose configs to support server start-up

### PR DESCRIPTION
**Pull request by caioriva**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,13 +8,14 @@
...
-    "start": "cross-env NODE_ENV=production node dist/index.js",
+    "start:development": "cross-env NODE_ENV=development node dist/src/index.js",
+    "start:production": "cross-env NODE_ENV=production node dist/src/index.js",
     "migrate-prisma": "npx prisma migrate dev --name init",
     "seed": "npx prisma db seed",
     "db-push": "npx prisma db push"
   },
   "keywords": [],
-  "author": "Cesar Riva",
+  "author": "Cesar Riva & Caio Riva",
```
Comment (cesarriva) [inline package.json:18]: Fair enough

**Comment by caioriva** [comment]
@cesarriva try to run it locally with the README I created and see if it's easy to follow.

You can use the following values for the .env file:
```
NODE_ENV=development
PORT=4000
API_ROOT=/api
JWT_SECRET=2bb15463-d719-4059-8c53-71c4092a0625
JWT_EXPIRES_IN=15m
JWT_REFRESH_SECRET=343be485-37da-4a06-8d56-4564dacd1e64
JWT_REFRESH_EXPIRES_IN=7d
DATABASE_USER=postgres
DATABASE_PASS=postgres
DATABASE_PORT=5432
DATABASE_NAME=mimic-mania
DATABASE_URL="postgresql://${DATABASE_USER}:${DATABASE_PASS}@host.docker.internal:${DATABASE_PORT}/${DATABASE_NAME}"
DATABASE_CLIENT_EMAIL=admin@email.com
DATABASE_CLIENT_PASS=admin
```

**Comment by caioriva** [comment]
@cesarriva also, test it with the frontend app. I haven't done it so it's good to double check especially that starting the server with docker-compose sets the DB schema.

**Review by cesarriva — APPROVED** [review]
LGTM, good stuff. awesome

**Pull request merged**